### PR TITLE
Setup Google Cloud integration for Firebase

### PR DESCRIPTION
**Pull request by TomasMorton**
- Adds terraform for Infrastructure
- Adds docs about how to setup

### CODE REVIEW

File: Infrastructure/firestore.rules [hunk 1]
```diff
@@ -0,0 +1,10 @@
+rules_version = '2';
+service cloud.firestore {
+  match /databases/{database}/documents {
+    match /{document=**} {
+      allow read: if request.auth != null;
+      allow create: if request.auth != null;
+      allow update: if request.auth != null;
```
Comment (TomasMorton) [inline Infrastructure/firestore.rules:7]: Will update these later, as we'll want more specific rules

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # ludum-dare-template
 
-A template prject used primarily for ludum dare gamejams. 
+A template project used primarily for ludum dare gamejams.
```
Comment (baconeta) [inline README.md:3]: Hehehe. I fixed this but never committed it. Idk why

**Review by baconeta — APPROVED** [review]

**Comment by baconeta** [comment]
Nothing here seems like it would cause any confusion or issues and it will help if we decide to integrate any firestore tools later

**Comment by TomasMorton** [comment]
> Nothing here seems like it would cause any confusion or issues and it will help if we decide to integrate any firestore tools later

Did you try it out?

**Pull request merged**